setup.py

```diff
@@ -462,7 +462,7 @@ def write_version(filename: str = str(AIRFLOW_SOURCES_ROOT / "airflow" / "git_ve
 
 _devel_only_tests = [
     "aioresponses",
-    "backports.zoneinfo>=0.2.1;python_version<'3.9'",
+    "backports.zoneinfo>=0.2.1",
```

Comment:
https://github.com/pganssle/zoneinfo#installation-and-depending-on-this-library

> Support for backports.zoneinfo in Python 3.9+ is currently minimal, since it is expected that you would use the standard library zoneinfo module instead

Reply:
Ah, so we need to exclude it indeed if PY > 3.9 and also not make it part of the serializers then